tests/osaka/eip7692_eof_v1/eip5450_stack/test_code_validation.py

```diff
@@ -172,7 +176,8 @@ def section_code_with(
     Also returns some traits of the section: `has_invalid_back_jump`, `rjump_snippet_pops`,
     `rjump_snippet_pushes`, `rjump_falls_off_code`
     """
-    code = Bytecode(min_stack_height=inputs, max_stack_height=inputs)
```

Comment:
for the curious: this code was part of a fix for `invalid_max_stack_height` problem, and that fix worked (#833), but it turns out these params weren't necessary and they actually don't have any effect.

But the added line below is necessary now and it does have the expected effect :shrug: